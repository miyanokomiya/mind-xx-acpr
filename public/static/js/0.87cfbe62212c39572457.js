webpackJsonp([0],{"9r5q":function(t,e,n){t.exports=n.p+"static/img/demo1.c26dfac.gif"},QEP9:function(t,e,n){(t.exports=n("FZ+f")(!0)).push([t.i,"\n.display-1[data-v-40074dd2] {\n  background-color: rgba(0, 0, 0, 0.3);\n  padding: 6px 12px;\n  border-radius: 2px;\n}\n","",{version:3,sources:["/Users/komiyamatomoya/develop/web/mind-xx-acpr/src/components/organisms/TopPage.vue"],names:[],mappings:";AACA;EACE,qCAAqC;EACrC,kBAAkB;EAClB,mBAAmB;CACpB",file:"TopPage.vue",sourcesContent:["\n.display-1[data-v-40074dd2] {\n  background-color: rgba(0, 0, 0, 0.3);\n  padding: 6px 12px;\n  border-radius: 2px;\n}\n"],sourceRoot:""}])},dVHn:function(t,e,n){var a=n("QEP9");"string"==typeof a&&(a=[[t.i,a,""]]),a.locals&&(t.exports=a.locals);(0,n("rjj0").default)("359e8472",a,!0,{})},eXG1:function(t,e,n){"use strict";Object.defineProperty(e,"__esModule",{value:!0});var a=n("9r5q"),r=n.n(a),o={computed:{Image:function(){return r.a}}},s=n("XyMi");var i=function(t){n("dVHn")},c=Object(s.a)(o,function(){var t=this.$createElement,e=this._self._c||t;return e("div",{staticStyle:{width:"100%"}},[e("section",[e("img",{staticStyle:{width:"100%",height:"auto","max-width":"600px"},attrs:{src:this.Image}})]),this._v(" "),e("section",[e("v-layout",{staticClass:"white--text",attrs:{column:"","align-center":"","justify-center":""}},[e("router-link",{staticStyle:{margin:"16px 0 24px 0","text-decoration":"none"},attrs:{to:{name:"WorkSpace"}}},[e("v-btn",{staticClass:"blue lighten-2",attrs:{dark:"",large:""}},[this._v("\n            Get Started\n          ")])],1)],1)],1)])},[],!1,i,"data-v-40074dd2",null);e.default=c.exports}});
//# sourceMappingURL=0.87cfbe62212c39572457.js.map