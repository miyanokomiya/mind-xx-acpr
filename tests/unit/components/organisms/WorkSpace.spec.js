import { mount, shallowMount } from '@vue/test-utils'
import Vuetify from 'vuetify'
import Target from '@/components/organisms/WorkSpace.vue'
import * as model from '@/utils/model.js'

const vuetify = new Vuetify()

describe('components/organisms/WorkSpace.vue', () => {
  describe('snapshot', () => {
    const user = model.createUser({ uid: 'u' })

    const file_a = model.createFile({ name: 'file_a', nodeCount: 1 })
    const file_b = model.createFile({ name: 'file_b', nodeCount: 2 })
    const file_c = model.createFile({ name: 'file_c', nodeCount: 2 })

    it('default', () => {
      const wrapper = mount(Target, {
        propsData: {
          user,
          files: { a: file_a, file_c },
          fileAuthorities: { a: model.createFileAuthority({ public: { write: true } }) },
          sharedFiles: { b: file_b },
          sharedFileAuthorities: { b: model.createFileAuthority() },
        },
        vuetify,
      })
      expect(wrapper.element).toMatchSnapshot()
    })
    it('shared', async () => {
      const wrapper = mount(Target, {
        propsData: {
          user,
          files: { a: file_a },
          fileAuthorities: { a: model.createFileAuthority() },
          sharedFiles: { b: file_b },
          sharedFileAuthorities: {
            b: model.createFileAuthority({
              users: { u: { write: true } },
            }),
          },
        },
        vuetify,
      })
      await wrapper.setData({ dataKind: 'shared' })
      expect(wrapper.element).toMatchSnapshot()
    })
    it('workSpace', async () => {
      const wrapper = mount(Target, {
        propsData: {
          user,
          files: { a: file_a },
          fileAuthorities: { a: model.createFileAuthority() },
          sharedFiles: { b: file_b },
          sharedFileAuthorities: { b: model.createFileAuthority() },
        },
        vuetify,
      })
      await wrapper.setData({ dataKind: 'workSpace' })
      expect(wrapper.element).toMatchSnapshot()
    })
  })

  describe('methods', () => {
    describe('changeName', () => {
      it('emit', () => {
        const file_a = model.createFile({ name: 'file_a', nodeCount: 1 })
        const wrapper = shallowMount(Target, {
          propsData: {
            files: { a: file_a },
          },
        })
        wrapper.vm.changeName({ key: 'a', name: 'new name' })
        expect(wrapper.emitted('changeName')).toEqual([
          [
            {
              files: {
                a: {
                  ...file_a,
                  name: 'new name',
                },
              },
            },
          ],
        ])
      })
    })

    describe('createFile', () => {
      it('emit', () => {
        const wrapper = shallowMount(Target)
        wrapper.vm.createFile()
        expect(wrapper.emitted('createFile')).toEqual([
          [
            {
              file: { name: '' },
            },
          ],
        ])
        expect(wrapper.vm.pagination.sortBy).toBe('updated')
        expect(wrapper.vm.pagination.descending).toBe(true)
      })
    })

    describe('deleteFile', () => {
      it('emit', () => {
        const wrapper = shallowMount(Target)
        expect(wrapper.vm.snackbar).toBe(false)
        wrapper.vm.deleteFile('a')
        expect(wrapper.vm.snackbar).toBe(true)
        wrapper.vm.deleteFile('a')
        expect(wrapper.emitted('deleteFiles')).toEqual([
          [
            {
              files: { a: null },
            },
          ],
        ])
        expect(wrapper.vm.snackbar).toBe(false)
      })
    })

    describe('cloneFile', () => {
      it('emit', () => {
        const file_a = model.createFile({ name: 'file_a', nodeCount: 1 })
        const wrapper = shallowMount(Target, {
          propsData: {
            files: { a: file_a },
          },
        })
        wrapper.vm.cloneFile('a')
        expect(wrapper.emitted('cloneFile')).toEqual([[{ fileKey: 'a' }]])
      })
    })
  })
})
