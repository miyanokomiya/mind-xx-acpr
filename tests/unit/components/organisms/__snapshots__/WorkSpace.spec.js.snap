// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/organisms/WorkSpace.vue snapshot default 1`] = `
<div
  class="work-space-component"
>
  <div
    class="v-card v-sheet theme--light"
  >
    <div
      class="v-card__title"
    >
      <div
        class="flex xs6"
      >
        <div
          class="v-input data-kind-select v-input--hide-details v-input--is-label-active v-input--is-dirty theme--light v-text-field v-text-field--single-line v-select"
        >
          <div
            class="v-input__control"
          >
            <div
              aria-expanded="false"
              aria-haspopup="listbox"
              aria-owns="list-4"
              class="v-input__slot"
              role="button"
            >
              <div
                class="v-select__slot"
              >
                <div
                  class="v-select__selections"
                >
                  <div
                    class="v-select__selection v-select__selection--comma"
                  >
                    All files
                  </div>
                  <input
                    aria-readonly="true"
                    bottom=""
                    id="input-4"
                    readonly="readonly"
                    type="text"
                  />
                </div>
                <div
                  class="v-input__append-inner"
                >
                  <div
                    class="v-input__icon v-input__icon--append"
                  >
                    <i
                      aria-hidden="true"
                      class="v-icon notranslate mdi mdi-menu-down theme--light"
                    />
                  </div>
                </div>
              </div>
              <div
                class="v-menu"
              >
                <div
                  class="v-menu__content theme--light "
                  style="max-height: 304px; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top left; z-index: 0; display: none;"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <div
        class="header-buttons"
      >
        <button
          class="v-btn v-btn--contained v-btn--fab v-btn--round theme--dark v-size--small green"
          type="button"
        >
          <span
            class="v-btn__content"
          >
            <i
              aria-hidden="true"
              class="v-icon notranslate material-icons theme--dark"
            >
              add
            </i>
          </span>
        </button>
      </div>
    </div>
  </div>
   
  <div
    class="v-card v-sheet theme--light"
  >
    <div
      class="header-tools"
    />
     
    <div
      class="flex xs-12"
    >
      <div
        class="file-card v-card v-sheet theme--light"
      >
        <div
          class="v-card__title card-title"
        >
          <div
            class="content-box"
          >
            <p
              class="file-name mb-0"
            >
              file_b
            </p>
             
            <dl>
              <dt>
                Nodes:
              </dt>
               
              <dd>
                2
              </dd>
            </dl>
             
            <dl>
              <dt>
                Updated:
              </dt>
               
              <dd>
                1970/01/01 09:00:00
              </dd>
            </dl>
          </div>
           
          <div
            class="button-box"
          >
            <span
              class="shared-tag v-chip v-chip--no-color theme--light v-size--default"
            >
              <span
                class="v-chip__content"
              >
                
              shared
            
              </span>
            </span>
             
            <div
              class="v-menu v-small-dialog theme--light hidden"
              lazy=""
            >
              <div
                class="v-small-dialog__activator"
              >
                <span
                  class="v-small-dialog__activator__content"
                >
                  <button
                    class="file-button edit-name v-btn v-btn--contained v-btn--fab v-btn--round theme--dark v-size--small blue"
                    type="button"
                  >
                    <span
                      class="v-btn__content"
                    >
                      <i
                        aria-hidden="true"
                        class="v-icon notranslate material-icons theme--dark"
                      >
                        textsms
                      </i>
                    </span>
                  </button>
                   
                </span>
              </div>
              <div
                class="v-menu__content theme--light v-small-dialog__menu-content"
                role="menu"
                style="max-height: auto; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top right; z-index: 0; display: none;"
              />
            </div>
             
            <button
              class="file-button v-btn v-btn--depressed v-btn--fab v-btn--flat v-btn--outlined v-btn--round theme--dark v-size--small black--text"
              type="button"
            >
              <span
                class="v-btn__content"
              >
                <i
                  aria-hidden="true"
                  class="v-icon notranslate material-icons theme--dark"
                >
                  content_copy
                </i>
              </span>
            </button>
             
            <button
              class="file-button v-btn v-btn--depressed v-btn--fab v-btn--flat v-btn--outlined v-btn--round theme--dark v-size--small black--text hidden"
              type="button"
            >
              <span
                class="v-btn__content"
              >
                <i
                  aria-hidden="true"
                  class="v-icon notranslate material-icons theme--dark"
                >
                  delete
                </i>
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="flex xs-12"
    >
      <div
        class="file-card v-card v-sheet theme--light"
      >
        <div
          class="v-card__title card-title"
        >
          <div
            class="content-box"
          >
            <p
              class="file-name mb-0"
            >
              file_a
            </p>
             
            <dl>
              <dt>
                Nodes:
              </dt>
               
              <dd>
                1
              </dd>
            </dl>
             
            <dl>
              <dt>
                Updated:
              </dt>
               
              <dd>
                1970/01/01 09:00:00
              </dd>
            </dl>
          </div>
           
          <div
            class="button-box"
          >
            <span
              class="shared-tag v-chip v-chip--no-color theme--light v-size--default hidden"
            >
              <span
                class="v-chip__content"
              >
                
              shared
            
              </span>
            </span>
             
            <div
              class="v-menu v-small-dialog theme--light"
              lazy=""
            >
              <div
                class="v-small-dialog__activator"
              >
                <span
                  class="v-small-dialog__activator__content"
                >
                  <button
                    class="file-button edit-name v-btn v-btn--contained v-btn--fab v-btn--round theme--dark v-size--small blue"
                    type="button"
                  >
                    <span
                      class="v-btn__content"
                    >
                      <i
                        aria-hidden="true"
                        class="v-icon notranslate material-icons theme--dark"
                      >
                        textsms
                      </i>
                    </span>
                  </button>
                   
                </span>
              </div>
              <div
                class="v-menu__content theme--light v-small-dialog__menu-content"
                role="menu"
                style="max-height: auto; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top right; z-index: 0; display: none;"
              />
            </div>
             
            <button
              class="file-button v-btn v-btn--depressed v-btn--fab v-btn--flat v-btn--outlined v-btn--round theme--dark v-size--small black--text"
              type="button"
            >
              <span
                class="v-btn__content"
              >
                <i
                  aria-hidden="true"
                  class="v-icon notranslate material-icons theme--dark"
                >
                  content_copy
                </i>
              </span>
            </button>
             
            <button
              class="file-button v-btn v-btn--depressed v-btn--fab v-btn--flat v-btn--outlined v-btn--round theme--dark v-size--small black--text"
              type="button"
            >
              <span
                class="v-btn__content"
              >
                <i
                  aria-hidden="true"
                  class="v-icon notranslate material-icons theme--dark"
                >
                  delete
                </i>
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="flex xs-12"
    >
      <div
        class="file-card v-card v-sheet theme--light"
      >
        <div
          class="v-card__title card-title"
        >
          <div
            class="content-box"
          >
            <p
              class="file-name mb-0"
            >
              file_c
            </p>
             
            <dl>
              <dt>
                Nodes:
              </dt>
               
              <dd>
                2
              </dd>
            </dl>
             
            <dl>
              <dt>
                Updated:
              </dt>
               
              <dd>
                1970/01/01 09:00:00
              </dd>
            </dl>
          </div>
           
          <div
            class="button-box"
          >
            <span
              class="shared-tag v-chip v-chip--no-color theme--light v-size--default hidden"
            >
              <span
                class="v-chip__content"
              >
                
              shared
            
              </span>
            </span>
             
            <div
              class="v-menu v-small-dialog theme--light hidden"
              lazy=""
            >
              <div
                class="v-small-dialog__activator"
              >
                <span
                  class="v-small-dialog__activator__content"
                >
                  <button
                    class="file-button edit-name v-btn v-btn--contained v-btn--fab v-btn--round theme--dark v-size--small blue"
                    type="button"
                  >
                    <span
                      class="v-btn__content"
                    >
                      <i
                        aria-hidden="true"
                        class="v-icon notranslate material-icons theme--dark"
                      >
                        textsms
                      </i>
                    </span>
                  </button>
                   
                </span>
              </div>
              <div
                class="v-menu__content theme--light v-small-dialog__menu-content"
                role="menu"
                style="max-height: auto; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top right; z-index: 0; display: none;"
              />
            </div>
             
            <button
              class="file-button v-btn v-btn--depressed v-btn--fab v-btn--flat v-btn--outlined v-btn--round theme--dark v-size--small black--text"
              type="button"
            >
              <span
                class="v-btn__content"
              >
                <i
                  aria-hidden="true"
                  class="v-icon notranslate material-icons theme--dark"
                >
                  content_copy
                </i>
              </span>
            </button>
             
            <button
              class="file-button v-btn v-btn--depressed v-btn--fab v-btn--flat v-btn--outlined v-btn--round theme--dark v-size--small black--text"
              type="button"
            >
              <span
                class="v-btn__content"
              >
                <i
                  aria-hidden="true"
                  class="v-icon notranslate material-icons theme--dark"
                >
                  delete
                </i>
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
   
  <!---->
</div>
`;

exports[`components/organisms/WorkSpace.vue snapshot shared 1`] = `
<div
  class="work-space-component"
>
  <div
    class="v-card v-sheet theme--light"
  >
    <div
      class="v-card__title"
    >
      <div
        class="flex xs6"
      >
        <div
          class="v-input data-kind-select v-input--hide-details v-input--is-label-active v-input--is-dirty theme--light v-text-field v-text-field--single-line v-select"
        >
          <div
            class="v-input__control"
          >
            <div
              aria-expanded="false"
              aria-haspopup="listbox"
              aria-owns="list-54"
              class="v-input__slot"
              role="button"
            >
              <div
                class="v-select__slot"
              >
                <div
                  class="v-select__selections"
                >
                  <div
                    class="v-select__selection v-select__selection--comma"
                  >
                    Shared files
                  </div>
                  <input
                    aria-readonly="true"
                    bottom=""
                    id="input-54"
                    readonly="readonly"
                    type="text"
                  />
                </div>
                <div
                  class="v-input__append-inner"
                >
                  <div
                    class="v-input__icon v-input__icon--append"
                  >
                    <i
                      aria-hidden="true"
                      class="v-icon notranslate mdi mdi-menu-down theme--light"
                    />
                  </div>
                </div>
              </div>
              <div
                class="v-menu"
              >
                <div
                  class="v-menu__content theme--light "
                  style="max-height: 304px; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top left; z-index: 0; display: none;"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <div
        class="header-buttons"
      >
        <!---->
      </div>
    </div>
  </div>
   
  <div
    class="v-card v-sheet theme--light"
  >
    <div
      class="header-tools"
    />
     
    <div
      class="flex xs-12"
    >
      <div
        class="file-card v-card v-sheet theme--light"
      >
        <div
          class="v-card__title card-title"
        >
          <div
            class="content-box"
          >
            <p
              class="file-name mb-0"
            >
              file_b
            </p>
             
            <dl>
              <dt>
                Nodes:
              </dt>
               
              <dd>
                2
              </dd>
            </dl>
             
            <dl>
              <dt>
                Updated:
              </dt>
               
              <dd>
                1970/01/01 09:00:00
              </dd>
            </dl>
          </div>
           
          <div
            class="button-box"
          >
            <span
              class="shared-tag v-chip v-chip--no-color theme--light v-size--default"
            >
              <span
                class="v-chip__content"
              >
                
              shared
            
              </span>
            </span>
             
            <div
              class="v-menu v-small-dialog theme--light"
              lazy=""
            >
              <div
                class="v-small-dialog__activator"
              >
                <span
                  class="v-small-dialog__activator__content"
                >
                  <button
                    class="file-button edit-name v-btn v-btn--contained v-btn--fab v-btn--round theme--dark v-size--small blue"
                    type="button"
                  >
                    <span
                      class="v-btn__content"
                    >
                      <i
                        aria-hidden="true"
                        class="v-icon notranslate material-icons theme--dark"
                      >
                        textsms
                      </i>
                    </span>
                  </button>
                   
                </span>
              </div>
              <div
                class="v-menu__content theme--light v-small-dialog__menu-content"
                role="menu"
                style="max-height: auto; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top right; z-index: 0; display: none;"
              />
            </div>
             
            <button
              class="file-button v-btn v-btn--depressed v-btn--fab v-btn--flat v-btn--outlined v-btn--round theme--dark v-size--small black--text"
              type="button"
            >
              <span
                class="v-btn__content"
              >
                <i
                  aria-hidden="true"
                  class="v-icon notranslate material-icons theme--dark"
                >
                  content_copy
                </i>
              </span>
            </button>
             
            <button
              class="file-button v-btn v-btn--depressed v-btn--fab v-btn--flat v-btn--outlined v-btn--round theme--dark v-size--small black--text hidden"
              type="button"
            >
              <span
                class="v-btn__content"
              >
                <i
                  aria-hidden="true"
                  class="v-icon notranslate material-icons theme--dark"
                >
                  delete
                </i>
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
   
  <!---->
</div>
`;

exports[`components/organisms/WorkSpace.vue snapshot workSpace 1`] = `
<div
  class="work-space-component"
>
  <div
    class="v-card v-sheet theme--light"
  >
    <div
      class="v-card__title"
    >
      <div
        class="flex xs6"
      >
        <div
          class="v-input data-kind-select v-input--hide-details v-input--is-label-active v-input--is-dirty theme--light v-text-field v-text-field--single-line v-select"
        >
          <div
            class="v-input__control"
          >
            <div
              aria-expanded="false"
              aria-haspopup="listbox"
              aria-owns="list-92"
              class="v-input__slot"
              role="button"
            >
              <div
                class="v-select__slot"
              >
                <div
                  class="v-select__selections"
                >
                  <div
                    class="v-select__selection v-select__selection--comma"
                  >
                    My files
                  </div>
                  <input
                    aria-readonly="true"
                    bottom=""
                    id="input-92"
                    readonly="readonly"
                    type="text"
                  />
                </div>
                <div
                  class="v-input__append-inner"
                >
                  <div
                    class="v-input__icon v-input__icon--append"
                  >
                    <i
                      aria-hidden="true"
                      class="v-icon notranslate mdi mdi-menu-down theme--light"
                    />
                  </div>
                </div>
              </div>
              <div
                class="v-menu"
              >
                <div
                  class="v-menu__content theme--light "
                  style="max-height: 304px; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top left; z-index: 0; display: none;"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <div
        class="header-buttons"
      >
        <button
          class="v-btn v-btn--contained v-btn--fab v-btn--round theme--dark v-size--small green"
          type="button"
        >
          <span
            class="v-btn__content"
          >
            <i
              aria-hidden="true"
              class="v-icon notranslate material-icons theme--dark"
            >
              add
            </i>
          </span>
        </button>
      </div>
    </div>
  </div>
   
  <div
    class="v-card v-sheet theme--light"
  >
    <div
      class="header-tools"
    />
     
    <div
      class="flex xs-12"
    >
      <div
        class="file-card v-card v-sheet theme--light"
      >
        <div
          class="v-card__title card-title"
        >
          <div
            class="content-box"
          >
            <p
              class="file-name mb-0"
            >
              file_a
            </p>
             
            <dl>
              <dt>
                Nodes:
              </dt>
               
              <dd>
                1
              </dd>
            </dl>
             
            <dl>
              <dt>
                Updated:
              </dt>
               
              <dd>
                1970/01/01 09:00:00
              </dd>
            </dl>
          </div>
           
          <div
            class="button-box"
          >
            <span
              class="shared-tag v-chip v-chip--no-color theme--light v-size--default hidden"
            >
              <span
                class="v-chip__content"
              >
                
              shared
            
              </span>
            </span>
             
            <div
              class="v-menu v-small-dialog theme--light hidden"
              lazy=""
            >
              <div
                class="v-small-dialog__activator"
              >
                <span
                  class="v-small-dialog__activator__content"
                >
                  <button
                    class="file-button edit-name v-btn v-btn--contained v-btn--fab v-btn--round theme--dark v-size--small blue"
                    type="button"
                  >
                    <span
                      class="v-btn__content"
                    >
                      <i
                        aria-hidden="true"
                        class="v-icon notranslate material-icons theme--dark"
                      >
                        textsms
                      </i>
                    </span>
                  </button>
                   
                </span>
              </div>
              <div
                class="v-menu__content theme--light v-small-dialog__menu-content"
                role="menu"
                style="max-height: auto; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top right; z-index: 0; display: none;"
              />
            </div>
             
            <button
              class="file-button v-btn v-btn--depressed v-btn--fab v-btn--flat v-btn--outlined v-btn--round theme--dark v-size--small black--text"
              type="button"
            >
              <span
                class="v-btn__content"
              >
                <i
                  aria-hidden="true"
                  class="v-icon notranslate material-icons theme--dark"
                >
                  content_copy
                </i>
              </span>
            </button>
             
            <button
              class="file-button v-btn v-btn--depressed v-btn--fab v-btn--flat v-btn--outlined v-btn--round theme--dark v-size--small black--text"
              type="button"
            >
              <span
                class="v-btn__content"
              >
                <i
                  aria-hidden="true"
                  class="v-icon notranslate material-icons theme--dark"
                >
                  delete
                </i>
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
   
  <!---->
</div>
`;
